theseus/core/tests/test_theseus_function.py

```diff
@@ -29,3 +28,14 @@ def test_theseus_function_init():
             assert name == cost_function.name
 
     assert len(set(all_ids)) == len(all_ids)
+
+
+def test_no_copy_vars_check():
+    variables = [theseus.core.Variable(torch.ones(1, 1), name="var_1")]
```

Comment:
This might be a remnant of before, but we should switch to using `th.` like all other tests based on our convention.